app/src/main/java/com/example/tudeeapp/data/TaskServicesImpl.kt

```diff
@@ -31,7 +31,7 @@ class TaskServicesImpl(
     }
 
     override fun getAllCategories(): Flow<List<Category>> {
-        return categoryDao.getCategories()
+        return categoryDao.getAllCategories()
```

Comment:
Great job 
But I don't like the way of creating a sperata branch just for refactoring name of function!